composer.json

```diff
@@ -27,9 +27,7 @@
         "phpunit/phpunit": "4.*"
     },
     "suggest": {
-        "league/oauth2-client": "Needed for XOAUTH2 authentication",
-        "hayageek/oauth2-yahoo": "Needed to use XOAUTH2 with Yahoo",
-        "stevenmaguire/oauth2-microsoft": "Needed to use XOAUTH2 with Microsoft services"
+        "league/oauth2-client": "Needed for XOAUTH2 authentication"
```

Comment:
This is a dependency of all the specific implementations, so specifying it by itself isn't useful.